httpstan/cache.py

```diff
@@ -94,12 +96,10 @@ def dump_fit(fit_bytes: bytes, name: str) -> None:
         name: Stan fit name
         fit_bytes: LZ4-compressed messages associated with Stan fit.
     """
-    cache_path = appdirs.user_cache_dir("httpstan", version=httpstan.__version__)
     # fits are stored under their "parent" models
-    fits_path = os.path.join(*([cache_path] + name.split("/")[:-1]))
-    fit_filename = os.path.join(fits_path, f'{name.split("/")[-1]}.jsonlines.lz4')
-    os.makedirs(fits_path, exist_ok=True)
-    with open(fit_filename, mode="wb") as fh:
+    fit_filename = cache_directory() / (name + ".jsonlines.lz4")
```

Comment:
The previous version was a bit more explicit ("Explicit is better than implicit."). Also, can this use model_directory instead? In theory, the function should not need to know the cache directory. (The original was unskillful.)